### PR TITLE
chore: Enhance error description for MySQL test connection

### PR DESCRIPTION
**Pull request by jackcloudquery**
#### Summary

⚠️ **If you're contributing to a plugin please read this section of the [contribution guidelines](https://github.com/cloudquery/cloudquery/blob/main/CONTRIBUTING.md#open-core-vs-open-source) 🧑‍🎓 before submitting this PR** ⚠️

* Provide more meaningful descriptions for error codes when MySQL test connection fails
* Fix publishing cloud UI

### CODE REVIEW

File: plugins/destination/mysql/cloud-config-ui/src/App.tsx [hunk 1]
```diff
@@ -19,10 +19,19 @@ const useCloudAppMock =
   window.self === window.top;
 const DevWrapper = useCloudAppMock ? CloudAppMock : Fragment;
 // eslint-disable-next-line unicorn/prefer-module
-const devWrapperProps: any = useCloudAppMock ? require('./.env.json') : undefined;
+const { plugin, ...devWrapperProps }: any = useCloudAppMock ? require('./.env.json') : {};
+
+const pluginProps = useCloudAppMock
+  ? plugin
+  : {
+      team: process.env.REACT_APP_PLUGIN_TEAM,
```
Comment (cduflo) [inline plugins/destination/mysql/cloud-config-ui/src/App.tsx:27]: I think you'll have to add these env vars to the .github/workflows/dest_mysql.yml

**Review by cduflo — COMMENTED** [review]
Just one comment to avoid a deployment hiccup.

Looks great! These changes will make the transition to the new lib easier too 🚀

File: plugins/destination/mysql/cloud-config-ui/src/App.tsx (hunk 1 above)
Comment (jackcloudquery) [reply plugins/destination/mysql/cloud-config-ui/src/App.tsx:27]: Nice, forgot about that, thanks!

**Review by cduflo — APPROVED** [review]
don't think I can do an e2e in the preview until the github workflow lands, but looks good to me 👍

**Pull request merged**